firmware/updating-firmware.md

```diff
@@ -8,7 +8,7 @@ has_children: true
 {:.no_toc}
 This procedure will show how to build and configure the SlimeVR firmware, as well as how to upload it to your tracker.
 
-There are currently two ways of uploading your firmware. One is using platformio, and the other is the [online firmware flasher](https://slimevr-firmware-tool.futurabeast.com/). The online flasher is the most user friendly but doesn't work with Safari or Firefox. The platformio version is less user friendly, but is recommended if you want to get into the nitty gritty of developing the firmware. 
+There are currently two ways of uploading your firmware. One is using platformIO, and the other is the [online firmware flasher](https://slimevr-firmware-tool.futurabeast.com/). The online flasher is the most user friendly but doesn't work with Safari or Firefox. The platformIO version is less user friendly, but is recommended if you want to get into the nitty gritty of developing the firmware. 
```

Comment:
```suggestion
There are currently two ways of uploading your firmware. One is using PlatformIO, and the other is the [online firmware flasher](https://slimevr-firmware-tool.futurabeast.com/). The online flasher is the most user friendly but doesn't work with Safari or Firefox. The PlatformIO version is less user friendly, but is recommended if you want to get into the nitty gritty of developing the firmware. 
```

Reply:
You are suggesting changes to the part I removed?

Reply:
That doesn't look like that to me, she's suggesting to write `PlatformIO` with this casing

Reply:
Ahhh I didn't even see that at first, I only saw the change to caps of IO in the negative section, my apologies